packages/utils/src/getToken.ts

```diff
@@ -0,0 +1,15 @@
+import getAccountsFromLocalStorage from './getAccountsFromLocalStorage';
+
+/**
+ * Gets the current user's auth `token` for the active `loginid` from the `localStorage`.
+ */
+const getToken = (loginid: string) => {
```

Comment:
[Question]: wouldn't it be better if we add new util functions to @deriv/utils repo?

Reply:
makes sense, noted, will migrate with next update (I want to proceed possibly quickly to avoid growing conflicts)

Reply:
Afaik `@deriv-com/utils` is still a WIP, right? We're using utils functions from the `utils` package inside deriv-app at the moment.

Reply:
@nijil-deriv for V2, we can move to @deriv-com/utils and that should be okay as mentioned by Yashim in the FE catch up call.

Reply:
I can give it a go and see how it works ;)